ext4: remove extra define of ext4_new_blocks_old from mballoc.c

From: Mingming Cao <cmm@us.ibm.com>

The function prototype of ext4_new_blocks_old() is defined in ext4_fs.h,
so we don't need the extra function prototype in mballoc.c

Signed-off-by: Mingming Cao <cmm@us.ibm.com>
Signed-off-by: "Theodore Ts'o" <tytso@mit.edu>
---
 fs/ext4/mballoc.c |    2 --
 1 file changed, 2 deletions(-)

Index: linux-2.6.25-rc3/fs/ext4/mballoc.c
===================================================================
--- linux-2.6.25-rc3.orig/fs/ext4/mballoc.c	2008-03-03 15:52:26.000000000 -0800
+++ linux-2.6.25-rc3/fs/ext4/mballoc.c	2008-03-03 15:52:42.000000000 -0800
@@ -576,8 +576,6 @@ static void ext4_mb_store_history(struct
 
 static struct proc_dir_entry *proc_root_ext4;
 struct buffer_head *read_block_bitmap(struct super_block *, ext4_group_t);
-ext4_fsblk_t ext4_new_blocks_old(handle_t *handle, struct inode *inode,
-			ext4_fsblk_t goal, unsigned long *count, int *errp);
 
 static void ext4_mb_generate_from_pa(struct super_block *sb, void *bitmap,
 					ext4_group_t group);
